src/components/calculator/util/FormatPrecision.ts

```diff
@@ -1,115 +1,63 @@
-// Emits a comma if the digit at `orderOfMagnitude` should be proceeded by one.
-function commaIfNeeded(
-  orderOfMagnitude: number,
-  thousandsSeparator: string,
-): string {
-  return orderOfMagnitude > 0 && orderOfMagnitude % 3 === 0
-    ? thousandsSeparator
-    : ''
-}
-
 /**
  * Format points for display - fixed point with a set precision.
- * This is necessary because float.toPrecision will use exponential notation for large or small numbers.
+ * By default, show 2 sigfigs for numbers greater than 1 and 1 sigfig for
+ * numbers less than 1.
  */
 export function fixedPointPrecision(
   valOrNull: number | null,
   maxNumber = 1e6,
-  decimalsNearMax = false,
-  thousandsSeparator = ',',
+  minNumber = 1,
```

Comment:
This variable name seems misleading to me. `maxNumber` is the largest possible number microCOVID will ever display. `minNumber` appears to be the smallest number for which 2 decimal places will be shown?

Perhaps minNumberForTwoSigFigs? Or could we just always show two sigfigs?

Reply:
renamed minNumber -> oneSigFigThreshold.